src/Components/bookHub/BookCard.js

```diff
@@ -50,31 +51,32 @@ export default function BookCard({
   useEffect(getAuthor, []);
 
   return ( // horizontal scroll not implemented
-    <Card className="card" sx={{ maxWidth: 345 }}>
+    <div className="card" style={{ margin: inCarousel ? '0' : '30px 16px 10px 16px', display: 'flex', flexDirection: 'column' }}>
       <Link class="link" to={`/book/${id}`}>
```

Comment:
is the class here necessary? I think it's the one throwing the error. Can you check if the link still works without the specification?